src/Linters/HHClientLinter.hack

```diff
@@ -19,7 +18,31 @@ final class HHClientLinter implements Linter {
   use LinterTrait;
   use SuppressibleTrait;
 
-  const type TConfig = shape();
+  const type TConfig =
+    shape(?'ignore_except' => keyset<int>, ?'ignore' => keyset<int>);
+
+  const type TErrorCode = int;
+
+  <<__Memoize>>
+  private function isErrorCodeConfiguredToIgnore(): (function(this::TErrorCode): bool) {
+    $ignore_except = $this->config['ignore_except'] ?? null;
+    $ignore = $this->config['ignore'] ?? null;
+    if ($ignore is null) {
+      if ($ignore_except is null) {
+        return $_ ==> false;
+      } else {
```

Comment:
Style: remove `} else {` after a return, to reduce nesting and improve readability. Ditto for below.